pulpcore/pulpcore/apidocs/views.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import response, permissions
+from drf_openapi.views import SchemaView
+from drf_openapi.entities import OpenApiSchemaGenerator
+
+
+class DocView(SchemaView):
```

Comment:
I think this class could use a little bit of documentation explain why it exists, but otherwise this all looks good!